presto-hive/src/test/java/com/facebook/presto/hive/TestHivePageSink.java

```diff
@@ -122,6 +123,10 @@ public void testAllFormats()
                     if (codec == NONE || !codec.isSupportedStorageFormat(format)) {
                         continue;
                     }
+                    // No compression support needed for PAGEFILE
+                    if (format == PAGEFILE) {
```

Comment:
It is fine to do not add compression support in this PR. However we should add the compression. Writing temporary files without any compression might be too costly IOPS wise.